docs/standard/net-standard.md

```diff
@@ -110,51 +100,66 @@ In order to build .NET Standard libraries in Visual Studio, make sure you have [
+
+### .NET Standard problems
 
-.NET Standard is the replacement for [Portable Class Libraries (PCL)](./cross-platform/cross-platform-development-with-the-portable-class-library.md). .NET Standard improves on the experience of creating portable libraries by curating a standard BCL and establishing greater uniformity across .NET implementations as a result. A library that targets .NET Standard is a PCL or a ".NET Standard-based PCL". Existing PCLs are "profile-based PCLs".
+Here are some problems with .NET Standard that help explain why .NET 5 is the better way to share code across platforms and workloads:
 
-.NET Standard and PCL profiles were created for similar purposes but also differ in key ways.
+- Slowness to add new APIs
 
-Similarities:
+  .NET Standard was created as an API set that all .NET implementations would have to support, so there was a review process for proposals to add new APIs. The goal was to standardize only APIs that could be implemented in all current and future .NET platforms. The result was that if a feature missed a particular release, you might have to wait for a couple of years before it got added to a version of the Standard. Then you'd wait even longer for the new version of .NET Standard to be widely supported.
 
-- Define APIs that can be used for binary code sharing.
+  **Solution in .NET 5:** When a feature is implemented, it's already available for every .NET 5 app and library because the code base is shared. And since there's no difference between the API specification and its implementation, you're able to take advantage of new features much quicker than with .NET Standard.
 
-Differences:
+- Complex versioning
 
-- .NET Standard is a curated set of APIs, while PCL profiles are defined by intersections of existing platforms.
-- .NET Standard linearly versions, while PCL profiles do not.
-- PCL profiles represents Microsoft platforms while .NET Standard is platform-agnostic.
+  The separation of the API specification from its implementations results in complex mapping between API specification versions and implementation versions. This complexity is evident in the table shown earlier in this article and the instructions for how to interpret it.
 
-### PCL compatibility
+  **Solution in .NET 5:** There's no separation between a .NET 5.x API specification and its implementation. The result is a simplified TFM scheme. There's one TFM prefix for all workloads: `net5.0` is used for libraries, console apps, and web apps. The only variation is a suffix that specifies platform-specific APIs for Windows desktop apps: `net5.0-windows`. Thanks to this TFM naming convention, you can easily tell whether a given app can use a given library. No version number equivalents table like the one for .NET Standard is needed.
```

Comment:
There are other suffixes too, not just Windows. See https://docs.microsoft.com/en-us/dotnet/standard/frameworks#net-5-os-specific-tfms.